modules/services/yabai/default.nix

```diff
@@ -0,0 +1,107 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.yabai;
+
+  toYabaiConfig = opts:
+    concatStringsSep "\n" (mapAttrsToList
+      (p: v: "yabai -m config ${p} ${toString v}") opts);
+
+in
+
+{
+  options = {
+    services.yabai = with types; {
+      enable = mkOption {
```

Comment:
nit, I generally prefer to define options using their explicit attr path which makes it a bit easier to search

```suggestion
    services.yabai.enable = mkOption {
```

Reply:
Makes sense, I’ll make this change when I’m at a proper machine 📱